t/orderservice.t

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/perl
```

Comment:
I think that we must to refactor the tests in order to isolate the methods under test to the other classes.